### DISCUSSION TITLE
Chat about anything and everything here

### DISCUSSION THREAD

**Discussion by boxjan** [SDIBTACM/training#108] [General]
开了 training 的 dicussion, Chat about anything and everything here

**Comment by sdibt** [comment]
什么时候回烟台玩？

**Reply by boxjan** [reply]
五六月找个天气好的周末！

**Reply by sdibt** [reply]
好，恭候大驾。

**Comment by boxjan** [comment]
不至于，就随便玩玩

**Comment by boxjan** [comment]
这周周报咕咕，因为周天上班
下周五一大概也咕咕
五一后的周末也可能是咕的
一切责任全在调休

**Comment by xiaojj** [comment]
今天写了脚本，每年12月31日将每个人这一年的周报汇总成一个文件。

**Reply by boxjan** [reply]
那肯定要拼内容啦

**Comment by wilsonjuxta** [comment]
看了我也想回去泡实验室训练了

**Comment by boxjan** [comment]
那约个时间一起？From: Tamara-0 ***@***.***>Sent: Friday, June 2, 2023, 17:43To: SDIBTACM/trainingCc: boxjan; AuthorSubject: Re: [SDIBTACM/training] Chat about anything and everything here (Discussion #108)
看了我也想回去泡实验室训练了

**Comment by xiaojj** [comment]
来吧，聘你们两个当金牌教练。
________________________________
发件人: boxjan ***@***.***>
发送时间: Friday, June 2, 2023 6:25:16 PM
收件人: SDIBTACM/training ***@***.***>
抄送: xiaojj ***@***.***>; Comment ***@***.***>
主题: Re: [SDIBTACM/training] Chat about anything and everything here (Discussion #108)


那约个时间一起？From: Tamara-0 ***@***.***>Sent: Friday, June 2, 2023, 17:43To: SDIBTACM/trainingCc: boxjan; AuthorSubject: Re: [SDIBTACM/training] Chat about anything and everything here (Discussion #108)
看了我也想回去泡实验室训练了

―Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

―
